perf.groovy

```diff
@@ -18,7 +18,7 @@ def static getBuildJobName(def configuration, def os) {
 [true, false].each { isPR ->
     ['Windows_NT'].each { os ->
       ['Release'].each { config ->
-        ['x64', 'x86'].each { arch ->
+        ['x64'].each { arch ->
```

Comment:
What's the reason for getting rid of x86?

Reply:
@dsplaisted 
The SDK repo [master] and Arcade does not seem to support running x86 tests - at least not with ease.